ipaplatform/base/paths.py

```diff
@@ -48,6 +48,8 @@ class BasePathNamespace(object):
     HTTPD_IPA_CONF = "/etc/httpd/conf.d/ipa.conf"
     HTTPD_NSS_CONF = "/etc/httpd/conf.d/nss.conf"
     HTTPD_SSL_CONF = "/etc/httpd/conf.d/ssl.conf"
+    HTTPD_CERT_FILE = "/etc/pki/tls/certs/httpd.crt"
```

Comment:
Are these the default file names for mod_ssl? Although we highly recommend to install IPA on its own machine, people tend to mix installation. I wonder if we should consider or own file names, e.g. ``ipa-httpd``.

Reply:
Sure, it's an easy fix and now is the time to doit.